platforms/core-configuration/file-collections/src/test/groovy/org/gradle/api/internal/file/collections/DefaultConfigurableFileCollectionSpec.groovy

```diff
@@ -598,21 +598,21 @@ class DefaultConfigurableFileCollectionSpec extends FileCollectionSpec {
 
```

Comment:
Minor tweaks to messages (which are now shared between CFCs and `AbstractProperty`)